include/mbgl/storage/default_file_source.hpp

```diff
@@ -144,6 +144,14 @@ class DefaultFileSource : public FileSource {
      */
     void deleteOfflineRegion(OfflineRegion&&, std::function<void (std::exception_ptr)>);
 
+    /*
+     * Invalidate all the tiles from an offline region forcing Mapbox GL to revalidate
+     * the tiles with the server before using. This is more efficient than deleting the
+     * offline region and downloading it again because if the data on the cache matches
+     * the server, no new data gets transmitted.
+     */
+    void invalidateOfflineRegion(OfflineRegion&, std::function<void (std::exception_ptr)>);
```

Comment:
why mutable reference?

Reply:
I'm following the style on this file and I believe it is like this on other methods (like delete) because of const-correctness. It will effectively change the offline region even though it could be passed as const ref.